1-org/envs/shared/iam.tf

```
@@ -18,32 +18,50 @@
   Audit Logs - IAM
 *****************************************/
 
+locals {
+  enabling_data_logs = var.data_access_logs_enabled ? ["DATA_WRITE", "DATA_READ"] : []
+}
+
 resource "google_organization_iam_audit_config" "org_config" {
-  count   = var.data_access_logs_enabled && var.parent_folder == "" ? 1 : 0
+  count   = var.parent_folder == "" ? 1 : 0
   org_id  = var.org_id
   service = "allServices"
 
-  audit_log_config {
-    log_type = "DATA_READ"
-  }
-  audit_log_config {
-    log_type = "DATA_WRITE"
+  ###################################################################################################
+  ### Audit logs can generate costs, to know more about it,
+  ### check the official documentation: https://cloud.google.com/stackdriver/pricing#logging-costs
+  ### To know more about audit logs, you can find more infos
+  ### here https://cloud.google.com/logging/docs/audit/configure-data-access
+  ### To enable DATA_READ and DATA_WRITE audit logs, set `data_access_logs_enabled` to true
+  ####################################################################################################
+  dynamic "audit_log_config" {
+    for_each = local.enabling_data_logs
```

nit: this would let us use a single dynamic block
```suggestion
    for_each = setunion(local.enabling_data_logs, ["ADMIN_READ"])
```